test/AlgebraicGeometry/Schemes/IdealSheaves.jl

```diff
@@ -111,29 +111,3 @@ end
   @test pbII(U[2]) isa Ideal
   @test haskey(pbII.I.obj_cache, U[2])
 end
-
-@testset "primary decomposition of ideal sheaves" begin
```

Comment:
Can we rewrite these tests to use the new methods?

Reply:
The new methods are tested in the Cartier to Weil divisor tests, as I replaced the primary decomposition in weil_divisor.